pkg/kubelet/kuberuntime/logs/logs_test.go

```diff
@@ -18,6 +18,11 @@ package logs
 
 import (
 	"bytes"
+	"context"
+	"io/ioutil"
+	testing2 "k8s.io/cri-api/pkg/apis/testing"
```

Comment:
Name `testing2`,  sounds `apitest` is better?

Reply:
good catch. Goland chose that name for me and I didn't notice. I renamed it just now.